src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509.cpp

```diff
@@ -373,13 +414,22 @@ extern "C" int32_t AppleCryptoNative_X509ImportCertificate(uint8_t* pbData,
     if (pOSStatus != nullptr)
         *pOSStatus = noErr;
 
-    if (pbData == nullptr || cbData < 0 || pCertOut == nullptr || pIdentityOut == nullptr || pOSStatus == nullptr)
+    if (pbData == nullptr || cbData < 0 || pCertOut == nullptr || pIdentityOut == nullptr || pOSStatus == nullptr ||
+        exportable != !!exportable)
```

Comment:
Yeah, making 2 and beyond be invalid leaves us open to recycle that spot for a flags value later if needed.